### PR TITLE
Refs #3809 - Style/blocks cop enabled

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by shlomizadok** [comment]
:+1:

File: test/factories/host_related.rb [hunk 1]
```diff
@@ -216,16 +216,16 @@ def set_nic_attributes(host, attributes, evaluator)
           :subnet,
           overrides
         )
-      }
-      interfaces { [ FactoryGirl.build(:nic_primary_and_provision,
+      end
+      interfaces do [ FactoryGirl.build(:nic_primary_and_provision,
```
Comment (domcleal) [inline test/factories/host_related.rb]: Nitpick, just move this onto the next line, under the `do`

File: test/factories/host_related.rb [hunk 2]
```diff
@@ -183,9 +183,9 @@ def set_nic_attributes(host, attributes, evaluator)
 
     trait :with_puppet do
       environment
-      puppet_proxy { FactoryGirl.create(:smart_proxy,
+      puppet_proxy do FactoryGirl.create(:smart_proxy,
```
Comment (domcleal) [inline test/factories/host_related.rb]: ditto

File: test/factories/host_related.rb [hunk 3]
```diff
@@ -233,19 +233,19 @@ def set_nic_attributes(host, attributes, evaluator)
...
+      end
+      domain do
         FactoryGirl.create(:domain,
           :dns => FactoryGirl.create(:smart_proxy,
                     :features => [FactoryGirl.create(:feature, :dns)])
         )
-      }
-      interfaces { [ FactoryGirl.build(:nic_managed,
+      end
+      interfaces do [ FactoryGirl.build(:nic_managed,
```
Comment (domcleal) [inline test/factories/host_related.rb]: ditto

File: test/factories/host_related.rb [hunk 4]
```diff
@@ -255,13 +255,13 @@ def set_nic_attributes(host, attributes, evaluator)
     trait :with_tftp_orchestration do
       managed
       with_tftp_subnet
-      interfaces { [ FactoryGirl.build(:nic_managed,
+      interfaces do [ FactoryGirl.build(:nic_managed,
```
Comment (domcleal) [inline test/factories/host_related.rb]: ditto

File: test/factories/host_related.rb [hunk 5]
```diff
@@ -270,9 +270,9 @@ def set_nic_attributes(host, attributes, evaluator)
       association :compute_resource, :factory => :libvirt_cr
       domain
       interfaces { [ FactoryGirl.build(:nic_primary_and_provision) ] }
-      puppet_ca_proxy { FactoryGirl.create(:smart_proxy,
+      puppet_ca_proxy do FactoryGirl.create(:smart_proxy,
```
Comment (domcleal) [inline test/factories/host_related.rb]: ditto

File: test/factories/host_related.rb [hunk 6]
```diff
@@ -321,15 +321,15 @@ def set_nic_attributes(host, attributes, evaluator)
     trait :with_puppet_orchestration do
       architecture
       ptable
-      operatingsystem { FactoryGirl.create(:operatingsystem,
+      operatingsystem do FactoryGirl.create(:operatingsystem,
```
Comment (domcleal) [inline test/factories/host_related.rb]: etc!

**Comment by dLobatog** [comment]
@domcleal Should be fixed now, I blame `--auto-correct`...

**Comment by domcleal** [comment]
Merged as 7deaced, thanks @eLobato.

**Pull request closed without merging**